src/app/zap-templates/templates/chip/helper.js

```diff
@@ -390,6 +354,20 @@ function chip_attribute_list_entryTypes(options)
   return templateUtil.collectBlocks(this.items, options, this);
 }
 
+/**
+ * Returns if a given command argument chip type has configurable change
+ *
+ * This function is meant to be used inside a {{#chip_*}} block.
+ * It will throws otherwise.
+ *
+ * @param {*} options
+ */
+function hasConfigurableChange()
+{
+  checkIsChipType(this, 'hasConfigurableChange');
+  return this.isDiscrete === false && this.isString === false;
```

Comment:
This does not seem like the right condition, per spec.  Per spec, I'd expect an `isAnalog` test here.  Do we not have that concept?  What is `isDiscrete` set to for other spec types that are neither discrete nor analog (structs and lists, e.g.)?

Reply:
There is no `analog` thing into the zap database.  Looking at what we need to support it, I have started to update the `types.xml` definition into #7873. It has end up into a large shuffling of `src/app/zap-templates/zcl`...

Even with it, I still need to update ZAP itself, to digest the `analog` keyword properly.